src/main.rs

```diff
@@ -4,9 +4,9 @@ mod process;
 mod token;
 
 use anyhow::Result;
-use process::files;
+use process::process_files;
```

Comment:
clippy in aggressive mode will warn when an entity starts with the name of its module. Maybe the whole module needs a better name?

Reply:
IF you want to go ahead, I'll rename the module from `process` to `io` later